File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2018w/Command/src/Sportsmen.java
public class Sportsmen 
{
   private final String name;
   public Sportsmen(String name)
   {
      this.name=name;
   }
   String swim() { return "swim"; }
   String run() { return "run"; }
   String jump() { return "jump"; }
   String execute(Command cmd) { return name+": "+cmd.execute(this); }
}

File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2021s/Id06TC1/Strategy/Command/Sportsmen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    public class Sportsmen
    {
        private string name;
        private StrategyRun run;
        public Sportsmen(string name,StrategyRun run) 
        {
            this.name=name; 
            this.run=run;
        }
        public String Run() { return run==null?"Running":run.execute(); }
        public String Jump() { return "Jumping"; }
        public String Swim() { return "Swimming"; }
        public String execute(SportsmenCommand cmd) { return name + ": " + cmd.execute(this); }
    }
}
